sass/govuk/elements/_components.scss

```diff
@@ -0,0 +1,24 @@
+// Box with turquoise background and white text
+// Used for transaction end pages, and Bank Holidays
+.box-highlight {
```

Comment:
I think it make sense to name files after the names of components so they can be individually imported.

To make it a standalone element it may need its dependencies imported:
```scss
@import 'govuk/-essentials';
````
See [other elements](https://github.com/mojular/govuk-elements/blob/master/sass/govuk/elements/_global-subheader.scss#L1) for examples.

And of course you should test it by importing just this element on a test page.